libass/ass_bitmap.h

```diff
@@ -102,7 +102,9 @@ bool realloc_bitmap(const BitmapEngine *engine, Bitmap *bm, int32_t w, int32_t h
 bool copy_bitmap(const BitmapEngine *engine, Bitmap *dst, const Bitmap *src);
 void ass_free_bitmap(Bitmap *bm);
 
-bool outline_to_bitmap(ASS_Renderer *render_priv, Bitmap *bm,
+struct render_context;
```

Comment:
If I’m not mistaken, this declaration is a no-op and is neither necessary nor useful. You can just use `struct render_context *state` without this line.

Reply:
I thought so to, but I got `warning: declaration of 'struct render_context' will not be visible outside of this function [-Wvisibility]` and `error: conflicting types for 'outline_to_bitmap'` when I did that. Maybe we're thinking of something C++-only?

Reply:
Huh. Very well then; I was indeed mistaken!

([This SO answer](https://stackoverflow.com/a/66929358/865331) provides the relevant quote from the standard.)